hadoop-hdds/container-service/src/main/resources/webapps/hddsDatanode/dn.js

```diff
@@ -36,20 +36,104 @@
                  volume.TotalCapacity = transform(volume.TotalCapacity);
                 })
                 });
+
+            $http.get("jmx?qry=Hadoop:service=HddsDatanode,name=SCMConnectionManager")
+                .then(function (result) {
+                    ctrl.dnmetrics2 = result.data.beans;
+                    ctrl.dnmetrics2.forEach(scm => {
```

Comment:
Nit: Rather than `dnmetrics2`, we can change this to something like `ctrl.heartbeatmetrics` or other more appropriate names.

Reply:
I will improve this part of the code.